README.md

```diff
@@ -39,56 +39,69 @@ Features
 
-4. Prior to starting the server, make sure that your socket (TCP or sock file) server is running for both the IN and OUT sockets. You will get an error if the JDT server cannot connect on your ports/files specified in the environment variables
 
-5. To start the server in the active terminal, run:
-```bash
-java -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=1044 -Declipse.application=org.eclipse.jdt.ls.core.id1 -Dosgi.bundles.defaultStartLevel=4 -Declipse.product=org.eclipse.jdt.ls.core.product -Dlog.level=ALL -noverify -Xmx1G -jar ./plugins/org.eclipse.equinox.launcher_1.5.200.v20180922-1751.jar -configuration ./config_linux -data /path/to/data --add-modules=ALL-SYSTEM --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED
-```
+Development First Time Setup
+----------------------------
 
-6. Choosing a value for `-configuration`: this is the path to your platform's configuration directory. For linux, use `./config_linux`. For windows, use `./config_win`. For mac/OS X, use `./config_mac`.
+**Pre-requisite: Java 11 must be installed on your machine and configured in Eclipse.**
 
-7. Choosing a value for `-data`: the value for your data directory, should be the directory where your active workspace is, and you wish for the java langserver to add in its default files. Should also be the absolute path to this directory, ie., /home/username/workspace
+0. Fork and clone the repository
+1. Install [Eclipse IDE for Eclipse Committers](https://www.eclipse.org/downloads/packages/release/2018-09/r/eclipse-ide-eclipse-committers) that will have the most needed plugins already installed. Alternatively,
+you can get the [Eclipse IDE for Java developers](https://www.eclipse.org/downloads/packages/release/2018-09/r/eclipse-ide-java-developers)
+and just install Eclipse PDE from the Eclipse Marketplace.
 
-8. Notes about debugging: the `-agentlib:` is for connecting a java debugger agent to the process, and if you wish to debug the server from the start of execution, set `suspend=y` so that the JVM will wait for your debugger prior to starting the server
+2. Once installed use `File > Open Projects from File System...` and
+point it at `eclipse.jdt.ls` and Eclipse should automatically
+detect the projects and import it properly.
+
+3. If, after importing the projects, you see an error on `pom.xml` about Tycho, you can use Quick Fix
+(Ctrl+1) to install the Tycho maven integration.
+
+4. At that point, some plug-ins should still be missing in order to build the project. You can either open `org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target` in the Target Editor (which is the default editor) and click on `Set Target Platform`, or alternatively, open `Preferences > Plug-in Development > Target Platform` and select `Java Language Server Target Definition`). Eclipse will take some time to download all the required dependencies. It should then be able to compile all the projects in the workspace.
```

Comment:
I'd also like to add instructions for how to develop eclipse.jdt.ls using eclipse.jdt.ls - I found https://github.com/eclipse/eclipse.jdt.ls/pull/305 but didn't really manage to extract how it is supposed to work.

Is https://github.com/gorkem/jdt.ls.pde/ still up2date and if so - should adding it to the `bundles` configuration enable eclipse.jdt.ls to load eclipse.jdt.ls ?

Reply:
Using eclipse.jdt.ls to dogfood eclispe.jdt.ls is already supported, please see the contribution guide https://github.com/eclipse/eclipse.jdt.ls/blob/master/CONTRIBUTING.md#a-setting-up-the-jdt-language-server-in-vs-code.

Also i think we can remove the part "Development First Time Setup" from README, because CONTRIBUTING.md has already covered how to setup the develop environment.

Reply:
I pushed a fixup that changes the Development First Time Setup section to just have a link to the contributing file.